src/categories.h

```diff
@@ -53,19 +54,43 @@ class CategoryFactory
   };
 
 public:
+  struct NexusCategory
+  {
+    NexusCategory(const QString& name, const int nexusID) : m_Name(name), m_ID(nexusID)
+    {}
+    QString m_Name;
```

Comment:
These should be private with getters. Do these need to be settable independently?

Reply:
While you're probably correct, I was more or less mirroring the older Category struct. If I change one I may as well change the other.